NICE.Identity.Management/ClientApp/src/views/User/User.tsx

```diff
@@ -1,20 +1,95 @@
-import React from "react";
+import React, { Component } from "react";
+
 import { RouteComponentProps, Link } from "react-router-dom";
+import { Breadcrumbs, Breadcrumb } from "@nice-digital/nds-breadcrumbs";
+import { Panel } from "@nice-digital/nds-panel";
+import { Tag } from "@nice-digital/nds-tag";
+
+import { UserType } from "../../models/types";
+import { Endpoints } from "../../data/endpoints";
+import { UnlockUser } from "../../components/UnlockUser/UnlockUser";
 
 type TParams = { id: string };
 
 type UserProps = {} & RouteComponentProps<TParams>;
 
-export const User = (props: UserProps) => {
-	const { match } = props;
-	const { params } = match;
-
-	return (
-		<div className="grid">
-			<div data-g="12">
-				<p>User: {params.id}</p>
-				<Link to={`/users/${params.id}/roles`}>Roles</Link>
-			</div>
-		</div>
-	);
+type UserState = {
+	data: Array<UserType>;
+	error: string;
 };
+
+export class User extends Component<UserProps, UserState> {
+	constructor(props: UserProps) {
+		super(props);
+
+		this.state = {
+			data: [],
+			error: "",
+		};
+	}
+
+	updateData = (updatedData: Array<UserType>) => {
+		this.setState({ data: updatedData });
+	};
+
+	fetchData = async (url: string) => {
```

Comment:
Worth pointing out here this using an arrow function as a class property is currently experimental syntax (see https://babeljs.io/docs/en/babel-plugin-proposal-class-properties), but enabled in CRA. It's fine to use, just be aware it's experimental I guess!